### PR TITLE
fix(user edit): fix user edit not being able to be saved

### PR DESCRIPTION
**Pull request by ankarhem**
The local user commit introduced a bug that made you unable to save edited users

also fixed another error that was on the user edit page

### CODE REVIEW

File: src/components/UserEdit/index.tsx [hunk 1]
```diff
@@ -80,6 +80,7 @@ const UserEdit: React.FC = () => {
         permissions: currentPermission,
         email: user?.email,
         avatar: user?.avatar,
+        userType: user?.userType,
```
Comment (sct) [inline src/components/UserEdit/index.tsx:83]: I don't think we should pass user type. It shouldn't be editable by the user anyways. We should instead change the user route on the server end to merge the actual editable values being submitted.

**Review by sct — CHANGES_REQUESTED** [review]

File: src/components/UserEdit/index.tsx (hunk 1 above)
Comment (ankarhem) [reply src/components/UserEdit/index.tsx:83]: Don't the same apply for email and avatar though?

Comment (sct) [reply src/components/UserEdit/index.tsx:83]: Avatar yes, email will be editable for local users. So no.

At some point we could also make avatars editable but not very important.

Comment (ankarhem) [reply src/components/UserEdit/index.tsx:83]: Hm ok. I see your point then.

The issue is not that it isn't merging the result I think, but that the User schema lists userType as a required parameter.

Comment (sct) [reply src/components/UserEdit/index.tsx:83]: Oh, yeah. Ok let's get this merged for now and we can refactor it later.

**Review by sct — APPROVED** [review]

**Pull request merged**